### ISSUE TITLE
Don't propose to delete upstream if it doesn't exist

### ISSUE THREAD

**Issue by jerone** [ForkIssues/TrackerWin#66]
When deleting a local branch, there is an option to delete the origin branch:
![image](https://user-images.githubusercontent.com/55841/49310370-63dac980-f4de-11e8-818c-eee9b1539450.png)

But this branch doesn't exists (anymore):
![image](https://user-images.githubusercontent.com/55841/49310669-2cb8e800-f4df-11e8-9288-70811929f3e9.png)

*Ps. The text doesn't fit.*

**Comment by Otiel** [comment]
I believe the check box is disabled, no?

I don't mind seeing this option disabled as it informs that the remote branch doesn't exist anymore.

**Comment by jerone** [comment]
The checkbox was not disabled.

I already deleted the mentioned branch, but I recreated the situation, checked it and I got the following message:
![image](https://user-images.githubusercontent.com/55841/49316051-fd5ea700-f4ef-11e8-8f3f-fd7c6dd19cef.png)


**Steps to reproduce:**
1. Create local branch.
2. Push branch to origin remote. Accept the tracking message.
3. Delete the remote origin branch.
4. Delete the local branch. Notice that there is an option to delete the (deleted) origin branch and that the checkbox in the modal is checkable.

*Edit: wrong error message screenshot.*
